keep/api/utils/tenant_utils.py

```diff
@@ -63,13 +199,87 @@ def create_api_key(
         role=role.get_name(),
     )
     session.add(new_installation_api_key)
+
     if commit:
         session.commit()
+
     logger.info(
         "Created API key",
         extra={"tenant_id": tenant_id, "unique_api_key_id": unique_api_key_id},
     )
-    return api_key
+
+    if is_system:
+        return api_key
+
+    else:
+        statement = (
+            select(TenantApiKey)
+            .where(TenantApiKey.tenant_id == tenant_id)
+            .where(TenantApiKey.reference_id == new_installation_api_key.reference_id)
+        )
+
+        new_api_key = session.exec(statement).first()
+        return {**vars(new_api_key), "secret": api_key}
```

Comment:
use new_api_key.dict() or just explicitly specify the attributes.
also - whats "secret"?

Reply:
thats meant to be the api key secret. Should i call it api_key_secret?